src/your-first-flutter-app-to-appstore/index.md

```diff
@@ -234,41 +248,40 @@ Now, moving back to the ***Codemagic web interface***: in your app workflow edit
 
```

Comment:
Commenting on the preceding paragraph. There's "Distribution section" vs it's "`Distribution` section" later in the doc